src/visualization/components/internal/LegendOrientation.tsx

```diff
@@ -52,83 +76,77 @@ interface Props extends VisualizationOptionProps {
- *  2.  opacity of the legend (ranges from 20% to 100%); default is 100%
- *  3.  row colorization:  does each row have colored text (the default), or does each row have a dot of color prefacing it, with
- *  text all the same default (foreground) color
- *
- *  The above is what the user sees.
- *
- *  what is actually happening:
- *
- *    1.  Legend Orientation.  Default is horizontal
- *          behind the scenes it is a 'rotation threshold'.  if the number of items is above the threshold, then
- *          the legend is displayed vertically (each line is a column); if the number of items is below the threshold, then the legend is
- *          displayed horizontally (each line is a row).  therefore, since the default is horizontal, the numbers being set are very high
- *          for horizontal (high enough that the database would thrash out/throttle before it got that high), or low (0 for vertical, since
- *          to display something it has to be higher than zero).  The default number itself has changed, so some users may see their graphs flip
- *          (before; without the feature flag enabled, the default was hard-coded to 10; so graphs with more than 10 lines of  data in the tooltip
- *          were vertical)
- *
- *    2.  Legend Opacity.  The actual number is between the minimum .2 and the maximum 1.0; but we are displaying it as a matter of percentage
- *    3.  Row Colorization.  This is the simplest.  just a true/false flag, it is true by default.
- */
-const LegendOrientation: FC<Props> = ({properties, update}) => {
-  const legendOpacity = properties?.legendOpacity
-  const legendOrientation = properties?.legendOrientationThreshold
+const eventPrefix = 'visualization.customize'
 
-  const handleSetOrientation = (threshold: number): void => {
-    update({
-      legendOrientationThreshold: threshold,
-    })
+const getToggleColor = (toggle: boolean): CSSProperties => {
+  if (toggle) {
+    return {color: InfluxColors.Cloud}
   }
+  return {color: InfluxColors.Sidewalk}
+}
 
-  const handleSetOpacity = (event: ChangeEvent<HTMLInputElement>): void => {
-    const value = convertUserInputToNumOrNaN(event)
-
-    if (isNaN(value) || value < LEGEND_OPACITY_MINIMUM) {
-      update({
-        legendOpacity: LEGEND_OPACITY_MAXIMUM,
-      })
-    } else {
-      update({
-        legendOpacity: value,
-      })
+const HoverLegendToggle: FC<HoverLegendToggleProps> = ({
+  legendHide,
+  handleSetHoverLegendHide,
+}) => {
+  const getHoverLegendHideStatus = (legendHide: boolean): string => {
+    if (legendHide) {
+      return 'Hide'
     }
+    return 'Show'
   }
 
-  const handleSetColorization = (): void => {
-    update({
-      legendColorizeRows: !properties.legendColorizeRows,
+  return (
+    <FlexBox
+      direction={FlexDirection.Row}
+      alignItems={AlignItems.Center}
+      margin={ComponentSize.Medium}
+      stretchToFitWidth={true}
+      style={{marginTop: '0.5em', marginBottom: '1.5em'}}
```

Comment:
can we set classNames here with some clockface variables as the margin?

Reply:
👍